src/appleseed-max-impl/appleseedinteractive/interactivesession.h

```diff
@@ -43,8 +43,10 @@
 #include <thread>
 
 // Forward declarations.
-namespace renderer { class Camera; }
-namespace renderer { class Project; }
+namespace foundation { class SearchPaths; }
```

Comment:
A forward declaration is actually not enough in this case, you'll need to include `foundation/utility/searchpaths.h`.

Reply:
I thought the same but it compiled and thought I still don't understand anything about c++ :)

Reply:
What most likely happens is that `SearchPaths` is already made available by an indirect `#include` statement, and the forward declaration is entirely redundant.